docs/Advanced-Usage/Conda.rst

```diff
@@ -0,0 +1,183 @@
+Non-Source Dependency Management
+================================
+
+In :doc:`/Initial-Setup/Setting-up-your-Manager-Instance`, we quickly copy-pasted the contents
+of ``scripts/machine-launch-script.sh`` into the EC2 Management Console and
+that script installed many dependencies that FireSim needs using
+`conda <https://conda.io/en/latest/index.html>`_,  a platform-agnostic package
+manager, specifically using packages from the `conda-forge community <https://conda-forge.org/#about>`_.
+
+In many situations, you may not need to know anything about conda.  By default, the
+``machine-launch-script.sh`` installs conda and all of the FireSim dependencies into ``/opt/conda``
+and adds the required setup to the system-wide ``/etc/profile.d/conda.sh`` init script to add
+``/opt/conda`` to everyone's path.
+
+However, the script is also flexible.  For example, if you do not have root access, you can specify
+an alternate install location with the ``--prefix`` option to ``machine-launch-script.sh``.  The only requirement
+is that you are able to write into the install location.  See ``machine-launch-script.sh --help`` for more details.
+
+.. warning::
+
+    Without root access you can do many things, like :doc:`/Building-a-FireSim-AFI`,
+    :ref:`meta-simulation` of a FireSim system using Verilator or even developing new features in FireSim.
+    However, to :ref:`run a simulation on a F1 FPGA <running_simulations>` , FireSim currently requires that
+    you are able to act as root via ``sudo``.
+
+Updating a Package Version
```

Comment:
Nit: Shouldn't `moto` be in backticks everywhere (i.e "\`moto\`")

Reply:
Looks like the same thing for "conda"

Reply:
Sure.  I'll update those.

Reply:
I think I got all these after Abe suggested it.